handsontable/src/tableView.js

```diff
@@ -345,14 +345,15 @@ class TableView {
 
       this.#mouseDown = false;
 
+      const isInsideDialog = rootWrapperElement?.querySelector('.ht-dialog')?.contains(event.target);
```

Comment:
It is not the best option to search the DOM for an element that belongs to the plugin to change the behavior of Core. Can I ask you to change that?